Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -82,6 +88,20 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager, ShiftSub
         previousRightTicks = rightDrive.getPosition();
         leftInchesTraveled = 0;
         rightInchesTraveled = 0;
+        
+        drivingStateLatch.addObserver((Observable o, Object arg) -> {
+            if(arg instanceof EdgeType) {
+                EdgeType edge = (EdgeType)arg;
+                String type = "";
+                if(edge == EdgeType.RisingEdge) {
+                    type = "starting";
+                }
+                else if(edge == EdgeType.FallingEdge) {
+                    type = "stopping";
+                }
+                log.info(String.format("Drive %s left: %.2f, right: %.2f", type, this.leftDrive.get(), this.rightDrive.get()));
```

Comment:
I agree that'll generally be the case for DriveWithJoysticks but for autonomous driving it seemed like it might be interesting to see what was being set in case things go whacky (even just the initial sign of things).